### ISSUE TITLE
Can't find the saved files

### ISSUE THREAD

**Issue by yazer79** [alexisrolland/docker-openssl#2]
Hi,
After cloning the repo, I successfully installed the docker  image with 
$ docker build -t docker-openssl:latest .

and then running successfully:
$ docker run -it --rm -v "/d/test:/openssl-certs" docker-openssl

runs fine and I generate the certs:
$ req -x509 -newkey rsa:4096 -nodes -out cert.pem -keyout key.pem -days 365
then $ exit to quit openSSL
When I browse to d:\test\openssl-certs, it's empty... 

Am I doing something wrong? or I misunderstood something?

OS: Windows 10 home running Docker community desktop

**Comment by alexisrolland** [comment]
Hello @yazer79 
Based on your description it should be working. Is it possible that you have permissions issues to write on this disk?

**Comment by yazer79** [comment]
No, I am the system admin .. I tried also running the cmd as admin and still no files are generated
does /d/test:/openssl-certs
equal to d:/test/openssl-certs?

**Comment by alexisrolland** [comment]
I'm not sure why it's not working. I have retested the following on my side (I'm on Windows Pro):
```
docker build -t docker-openssl:latest .
docker run -it --rm -v "C:\Users\alexis.rolland\docker-openssl:/openssl-certs" docker-openssl
req -x509 -newkey rsa:4096 -nodes -out cert.pem -keyout key.pem -days 365
```
It worked and I can see the certificates created in my repo.
I will retest later with Windows Home.

**Comment by alexisrolland** [comment]
Hello @yazer79 
Regarding your question:

> does /d/test:/openssl-certs equal to d:/test/openssl-certs?

Actually no. The string ` /d/test:/openssl-certs` means Docker is mapping your host folder (on your machine) `D:\test` with the folder in the Docker container `/openssl-certs`. This is to make sure every file created in the folder `/openssl-certs` is copied to your machine so that you can retrieve the files once you shut down the container.

I was able to reproduce your issue on Windows Home:
```
cd docker-openssl
docker build -t docker-openssl:latest .
docker run -it --rm -v "/c/test:/openssl-certs" docker-openssl
req -x509 -newkey rsa:4096 -nodes -out cert.pem -keyout key.pem -days 365
```

Once I shutdown the container, there was no `test` folder created in the C: drive. I do not know why this issue happens but after changing the host folder path to something else like below it worked:

```
docker run -it --rm -v "/c/Users/Alexis/Documents/Github/docker-openssl:/openssl-certs" docker-openssl
req -x509 -newkey rsa:4096 -nodes -out cert.pem -keyout key.pem -days 365
```

**Comment by yazer79** [comment]
I tried and not working
"/c/Users/Alexis/Documents/Github/docker-openssl"
Is this the path where you ran docker build?

**Comment by alexisrolland** [comment]
@yazer79 yes this is the path where I cloned the repository, I set it as current directory when I build and run the container. As demonstrated above, this is also the path I mount as a volume in the container.

Try to run the container without mounting the volume. Once you generate the certificate, check inside the container that the files are properly created /openssl-certs, this should at least help to narrow down the issue to see if it’s a problem with the certificate generation or with the Docker volume mounting.

Let me know
